### PR TITLE
update README to reflect params, techniques and add pipeline

### PR DESCRIPTION
**Pull request by szd55gds**
**What**
* Large update to provide some knowledge and experience.
* Introduce pipeline first, cos this is the important 'thing'.
* Some examples and explanations of scripts for manual intervention.
* Introduce the parameters used when selecting records.

**Why**
The README required an update to reflect the current automated solution.

(solo)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,83 @@
...
+#### Required 
+
+Download and install correct version of 'fly'
 
-### Installing ruby
+
+#### Login to selected Concourse
+
+```
+fly --target [target-name] login --team-name [team] --concourse-url [concourse-url]
```
Comment (blairboy362) [inline README.md]: Can you replace the templates with actual values? Better to give an exact example for the prod multi-tenant concourse, and if you feel it's necessary link people to the `fly login` documentation.

File: README.md [hunk 2]
```diff
@@ -1,22 +1,83 @@
...
+```
+
+
+#### Secure required secrets
+
+Run 'show-available-pipeline-variables' found inside the 'info' pipeline and follow instructions for the following secrets:
```
Comment (blairboy362) [inline README.md]: Do we need to clarify that this means hijacking the job? Should we include a recommendation to set the secrets per pipeline?

File: README.md (hunk 1 above)
Comment (szd55gds) [reply README.md]: Sure, I was concerned about putting real values in there ;-)

File: README.md (hunk 2 above)
Comment (szd55gds) [reply README.md]: I agree with both and will add.

**Pull request merged**